torch/testing/_internal/common_optimizers.py

```diff
@@ -50,6 +51,10 @@
 )
 
 
+def is_dynamo_enabled() -> bool:
+    return os.environ.get("PYTORCH_TEST_WITH_DYNAMO") == "1"
```

Comment:
the one lined function seems unnecessary

Reply:
I did this since it was used in multiple places, not sure if we'll ever change the env var name

Reply:
Then this should be a function in common_utils.py where the env var gets read, and not in common_optimizers.py

Reply:
Oh, and it already exists: TEST_WITH_TORCHDYNAMO